partner_multi_company/models/res_users.py

```diff
@@ -14,13 +14,13 @@ def create(self, vals):
         if 'company_ids' in vals:
             res.partner_id.company_ids = vals['company_ids']
         if 'company_id' in vals:
-            res.partner_id.company_ids = [(6, 0, [vals['company_id']])]
+            res.partner_id.company_ids = [(4, vals['company_id'])]
```

Comment:
Don't change this. It's on purpose.

Reply:
Would you mind explain why?

Reply:
Because we want to overwrite full list, not to add one value to the current list.

Reply:
Would you mind check mi last post and let me know what you think?